packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/CharacteristicsFilter/components/FilterButton.js

```diff
@@ -55,6 +55,7 @@ export default function FilterButton (props) {
       />
       {checked && (
         <CloseButton
+          aria-label={`Remove ${valueLabel} filter`}
```

Comment:
Should this have a translation string? 
If so, I'm thinking it can be added in a subsequent PR, but I'm not sure.

Reply:
My test for this: what would a French volunteer expect this button to say? Here, I’d expect the label to be French.

Reply:
We do have survey projects in Spanish, if you want to test translations. 

Chimp & See is probably available in German too.

Reply:
Yes I agree, thanks for comments and references!
It was a small change so I added with [551b04d](https://github.com/zooniverse/front-end-monorepo/pull/3842/commits/551b04d2b41f57554c82261bf6eaf075e01cb71b).
I'll follow [the related translations docs/steps](https://github.com/zooniverse/how-to-zooniverse/blob/master/Translations/lokalise.md#pulling-new-dictionary-keys-into-lokalise-from-github) after merge.

Reply:
Fixing related tests atm...